packages/marko-web-theme-monorail/components/nodes/section-feed-content.marko

```diff
@@ -17,6 +17,16 @@ $ const withSection = defaultValue(input.withSection, true);
 $ const withTeaser = defaultValue(input.withTeaser, true);
 $ const displayImage = defaultValue(input.displayImage, true);
 $ const lazyload = defaultValue(input.lazyload, true);
+// linkOffsite is an object like the following:
```

Comment:
@brandonbk  & @Shinsina opinion question on this one.  This will work or an alternative is allow for a link prop to be passed in which would set these from whichever component is call it 

This file would need this added instead
```markojs
<!-- set link values and attrs based on input -->
$ const link = {
  ...(linkAttrs &&  { attrs: linkAttrs }),
  ...(input.link && { ...getAsObject(input, "link") })
};
<!-- set href value for image based of above ink props -->
$ const href = (link && link.field) ? get(content, link.field) : get(content, "siteContext.path");
```

link={ field: linkField, attrs: linkAttrs } would become link=link
& 
<@link href=link attrs=linkAttrs /> would become something link <@link href=href attrs=linkAttrs />

The part this still doesn't address is the targeting of the link base on external vs internal, togging base on if linkUrl is set.

Reply:
@B77Mills https://github.com/parameter1/base-cms/pull/552/commits/45dc2af16fde7672a6530be48f4e779a09094a5a The changes here should allow for the toggling of `target="_blank"` per site, think it's important to assure this behavior is consistent for anywhere this component is utilized as that's ultimately the "goal" is to drive traffic to these links when configured so regardless of the site location it should use this behavior